projects/dex-ui/src/components/Swap/SwapRoot.tsx

```diff
@@ -320,12 +322,13 @@ export const SwapRoot = () => {
   const getLabel = useCallback(() => {
     if (!account) return "Connect Wallet";
     if (!inAmount && !outAmount) return "Enter Amount";
+    if (inToken.address === outToken.address) return "Select different output token";
```

Comment:
A better approach here would be to not even allow the user to select the same token. We could add the prop `excludeTokenFromPicker={otherSelectedToken}` to TokenInput, then pass that down to the TokenPicker to use to filter out of the list.